src/Annotations/Serializer/CommonMark/MarkdownSanitizer.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace eLife\Annotations\Serializer\CommonMark;
+
+use HTMLPurifier;
+use League\CommonMark\Converter;
+use League\HTMLToMarkdown\HtmlConverter;
+
+class MarkdownSanitizer
+{
+    private $htmlConverter;
+    private $htmlPurifier;
+    private $markdownConverter;
+
+    public function __construct(Converter $markdownConvertor, HtmlConverter $htmlConverter, HTMLPurifier $htmlPurifier)
+    {
+        $this->markdownConverter = $markdownConvertor;
+        $this->htmlConverter = $htmlConverter;
+        $this->htmlPurifier = $htmlPurifier;
+    }
+
+    public function parse(string $input) : string
```

Comment:
This API conversion pipeline needs its step to be listed as documentation in the README:
```
1. Hypothesis API to `Model` objects
2. Denormalization
2.1 Conversion to HTML (uses library X)
2.2 HTML purification (uses library Y)
...
```
(ADRs are not a good format for that)

Reply:
The obvious questions that needs to be answered there is why we sanitize intermediate HTML and convert back to Markdown rather than sanitizing the final HTML